### DISCUSSION TITLE
The perfect launcher doesn't exist...

### DISCUSSION THREAD

**Discussion by tonydero** [MrCreeper1008/SpotlightLauncher#39] [General]
At least that was the struggle I found myself in until I found Starlight.

I mean everything, down to the name, is amazing. Really looking forward to seeing where this goes!

**Comment by kennethnym** [comment]
Sorry if this went unnoticed, for some reason Github doesn't send me notifications when a new discussion is made.

That said, I really appreciate your kind word! The next update will be available soon, which includes a bunch of bug fixes and a new app drawer.

**Reply by tonydero** [reply]
No worries!
